src/validation.cpp

```diff
@@ -21,6 +21,7 @@
 #include <flatfile.h>
 #include <hash.h>
 #include <kernel/chainparams.h>
+#include <kernel/context.h>
```

Comment:
This also seems to come from a previous revision.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/28051#discussion_r1304835728

> This also seems to come from a previous revision.

Thanks, reverted